src/components/guideGrid.tsx

```diff
@@ -11,14 +11,14 @@ type Props = {
 
 export function GuideGrid({platform, className}: Props) {
   const [currentPlatform] = usePlatform(platform);
-  // platform might actually not be a platform, so lets handle that case gracefully
-  if (!(currentPlatform as Platform).guides) {
+
+  if (currentPlatform.type === 'guide') {
```

Comment:
We're now able to discriminate on the `type` property to indicate if it's a platform or a platform guide. This is a lot cleaner.